src/core/meshcop/tlvs.hpp

```diff
@@ -575,6 +575,15 @@ class SteeringDataTlv: public Tlv
     void Set(void) { memset(mSteeringData, 0xff, GetLength()); }
 
     /**
+     * Ths method indicates whether or not the SteeringData allows all Joiners.
+     *
+     * @retval TRUE   If the SteeringData allows all Joienrs.
+     * @retval FALSE  If the SteeringData doesn't allow any Joiner.
+     *
+     */
+    bool IsAllowAny(void) { return ((GetLength() == 1) && (mSteeringData[0] == 0xff)); }
```

Comment:
nit: `IsAllowAny` -> `DoesAllowAny`.

Also, with bloom filters, any steering data length with all `0xff`'s will allow all joiners.  Should probably loop through all bytes and see if all are `0xff` rather than assuming a 1-byte length.